pisa/core/bin_indexing.py

```diff
@@ -0,0 +1,243 @@
+"""
+Functions to retrieve the bin location of each elements
+of an array, inside a Container, based on its specified 
+output binning.
+
+Etienne Bourbeau
+
+functions were adapted from translation.py
+
+"""
+
+from __future__ import absolute_import, print_function, division
+
+import numpy as np
+from numba import guvectorize, SmartArray, cuda
+
+from pisa import FTYPE, TARGET
+from pisa.core.binning import OneDimBinning, MultiDimBinning
+from pisa.utils.log import logging, set_verbosity
+from pisa.utils.numba_tools import myjit, WHERE
+from pisa.utils import vectorizer
+
+__all__ = [
```

Comment:
should change to contain actual function of this file

Reply:
Should it just be the functions I want people to use (ie not the functions that are only called internally?

Reply:
right. I meant before it was containing function that were _not_ in the file (like `histogram`)